packages/css-framework/src/components/_scrollable-nav.scss

```diff
@@ -28,8 +28,13 @@ $btn-focus-width: 0.2rem;
   border-top: 4px solid transparent;
   border-bottom: 4px solid transparent;
   cursor: pointer;
+  position: relative;
```

Comment:
Whats the relative for?

Reply:
Also, make sure you merge from develop. There will be a conflict on this file I think.

Reply:
good spot, removed.